app/src/main/java/io/github/trojan_gfw/igniter/tile/ProxyHelper.java

```diff
@@ -40,9 +40,7 @@ public static boolean isVPNServiceConsented(Context context) {
     }
 
     public static void startProxyService(Context context) {
-        Intent intent = new Intent(context, ProxyService.class);
-        intent.putExtra(ProxyService.CLASH_EXTRA_NAME, MultiProcessSP.getEnableClash(true));
-        ContextCompat.startForegroundService(context, intent);
+        ContextCompat.startForegroundService(context, new Intent(context, ProxyService.class));
```

Comment:
So https://github.com/trojan-gfw/igniter/issues/200 is also solved.

Reply:
Exactly